rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -61,39 +66,157 @@
 
 logger = logging.getLogger()
 
+
+def get_mask_paths(
+        mask_input_dir,
+        tilespecs,
+        read_masks_from_lowres_stack,
+        exts=['png', 'tif']):
+
+    results = {}
+
+    if read_masks_from_lowres_stack:
+        for t in tilespecs:
+            if t.ip[0].maskUrl is not None:
+                results[t.tileId] = t.ip[0].maskUrl
+
+    elif mask_input_dir is not None:
+        tids = [t.tileId for t in tilespecs]
+        maskfiles = []
+        for ext in exts:
+            e = mask_input_dir + '/*.' + ext
+            maskfiles += glob.glob(e)
+        maskbasenames = np.array([os.path.basename(os.path.splitext(p)[0])
+                                  for p in maskfiles])
+        for t in tids:
+            ind = np.argwhere(maskbasenames == t).flatten()
```

Comment:
Is this an assumption that mask names represent the tileids' name?.

Reply:
already in apply rough schema:
```
    mask_input_dir = InputDir(
        required=False,
        default=None,
        missing=None,
        description=("directory containing mask files. basenames of "
                     "masks that match tileIds in the rough stack "
                     "will be handled."))
```